src/Tasks.js

```diff
@@ -52,7 +52,7 @@ const Tasks = ({ tasks, setTasks }) => {
     }).catch(err => console.log(err));
   };
 
-  const testFunction = (value, event) => {
+  const editChecked = (value, event) => {
     setChecked(event.isCheck)
```

Comment:
state it is asynchronous function. you don't have current data.
you change the state twice